#motor speed (range: 0-80)
float32 motorRightSpeed
float32 motorLeftSpeed
